WalletWasabi.Gui/Shell/Commands/HelpCommands.cs

```diff
@@ -85,7 +88,11 @@ public HelpCommands(CommandIconService commandIconService)
 				.Merge(DocsCommand.GetReactiveCommand().ThrownExceptions)
 				.Merge(LegalDocumentsCommand.GetReactiveCommand().ThrownExceptions)
 				.ObserveOn(RxApp.TaskpoolScheduler)
-				.Subscribe(ex => Logger.LogError(ex));
+				.Subscribe(ex =>
+				{
+					NotificationHelpers.Error(ex.ToUserFriendlyString());
+					Logger.LogError(ex);
```

Comment:
Log the error first then display the UI notification. https://github.com/zkSNACKs/WalletWasabi/pull/2997#issue-365999512

Reply:
Done.